content/docs/accessibility.md

```diff
@@ -330,6 +330,10 @@ Testing with a screen reader should form part of your accessibility tests.
 
 Please note that browser / screen reader combinations matter. It is recommended that you test your application in the browser best suited to your screen reader of choice.
 
+### Commonly Used Screen Readers
+
+Refer to the [WebAIM Screen Reader Survey](https://webaim.org/projects/screenreadersurvey7/#primary) for a listing of screen readers and their usages.
```

Comment:
I still have the feeling that we should drop this line from the doc. 

By just having the headings of **Commonly used** and **Other** I think we are guiding the devs with what is in the survey. 

The survey has a lot of info and captured at a specific point in time so even today it could already be different. I don't feel that the React Docs is the place for that, rather that we should be using the a11y doc page to guide devs with the information we have. 

Can you remove this line?